src/sdk/storage/entities/identityVerificationStorage.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable indent */
+import { Entity, Unique, Column, Index, PrimaryColumn } from 'typeorm';
+
+export enum VcStatus {
+    APPROVED = 'APPROVED',
+    PENDING = 'PENDING',
+    REJECTED = 'REJECTED',
+}
+
+@Entity('IdentityVerificationStorage')
+@Unique(['veriffId'])
+export class IdentityVerificationStorage {
+    @PrimaryColumn({
+        unique: true,
+        type: 'varchar',
+    })
+    @Index()
+    veriffId!: string;
+
+    @Column({ type: 'varchar' })
+    vc!: string;
+
+    @Column({
+        type: 'enum',
+        enum: VcStatus,
+        default: VcStatus.PENDING,
+    })
+    status!: string;
```

Comment:
we still need to add the `type` column in here (a string enum (kyc, firstName, lastName, birthDate...)